corehq/apps/app_manager/views/modules.py

```diff
@@ -1266,6 +1241,42 @@ def edit_module_detail_screens(request, domain, app_id, module_unique_id):
     return JsonResponse(resp)
 
 
+def _update_sort_elements(domain, lang, detail, sort_elements):
+    # Attempt to map new elements to old so we don't lose translations
+    # Imperfect because the same field may be used multiple times, or user may change field
+    old_elements_by_field = {}
+    old_elements_by_calculation = {}
+    for sort_element in detail.short.sort_elements:
+        if sort_element.sort_calculation:
+            old_elements_by_calculation[sort_element.sort_calculation] = sort_element
+        elif sort_element.field:
+            old_elements_by_field[sort_element.field] = sort_element
+    detail.short.sort_elements = []
+    for sort_element in sort_elements:
+        item = _init_new_sort_element(sort_element)
+        if toggles.SORT_CALCULATION_IN_CASE_LIST.enabled(domain):
+            item.sort_calculation = sort_element['sort_calculation']
```

Comment:
It looks like you're just cloning `sort_element`. What about ...
```suggestion
        item = SortElement(dict(sort_element))
        if not toggles.SORT_CALCULATION_IN_CASE_LIST.enabled(domain):
            item.sort_calculation = ''
```